README.tfdoc.hcl

```
@@ -0,0 +1,338 @@
+
+header {
+  image = "https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg"
+  url   = "https://mineiros.io/?ref=terraform-google-service"
+
+  badge "Terraform Version" {
+    image = "https://img.shields.io/badge/Terraform-1.x-623CE4.svg?logo=terraform"
+    url   = "https://github.com/hashicorp/terraform/releases"
+    text  = "Terraform Version"
+  }
+
+  badge "Google Provider Version" {
+    image = "https://img.shields.io/badge/google-3.x-1A73E8.svg?logo=terraform"
+    url   = "https://github.com/terraform-providers/terraform-provider-google/releases"
+    text  = "Google Provider Version"
+  }
+
+  badge "Join Slack" {
+    image = "https://img.shields.io/badge/slack-@mineiros--community-f32752.svg?logo=slack"
+    url   = "https://mineiros.io/slack"
+    text  = "Join Slack"
+  }
+}
+
+section {
+  title   = "terraform-google-service"
+  toc     = true
+  content = <<-END
```

fmt just needs to align all single line arguments.. 

```suggestion
  title = "terraform-google-service"
  toc   = true
  content = <<-END
```